src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -329,11 +349,42 @@ public async Task<bool> DeleteSecretAsync(string secretName, string keyScope, Sc
             }
         }
 
+        internal static ulong GetKeySeed(ScriptSecretsType secretsType, string keyScope)
```

Comment:
`SecretGenerator` has been modified to return a key type from a seed, would it make sense to move this method there as well so we have type -> seed and seed -> type handed in the same place?

Reply:
Done